ush/calcanl_gfs.py

```diff
@@ -22,7 +22,8 @@ def calcanl_gfs(DoIAU, l4DEnsVar, Write4Danl, ComOut, APrefix, ASuffix,
 
     IAUHH = IAUHrs
     if Cdump == "gfs":
-        IAUHH = list(map(int,'6'))
+        #IAUHH = list(map(int,'6'))
```

Comment:
I am not sure whether this change will affect others, so I keep the organize statement. After we make the final decision, I will clean this up.

Reply:
I think this may break IAU

Reply:
I guess this isn't needed, so it should be deleted.

Reply:
For this one, I think I can remove the "if" since else has the same statement with "if"